Documentation/ApiOverview/Seo/PageTitleApi.rst

```diff
@@ -64,11 +68,50 @@ this method you can create your own logic to define the correct title.
 
 Usage example, for example, in an :ref:`Extbase <extbase>` controller:
 
-.. code-block:: php
+..  code-block:: php
+    :caption: EXT:my_extension/Classes/Controller/SomeController.php
+
+    $titleProvider = GeneralUtility::makeInstance(MyOwnPageTitleProvider::class);
```

Comment:
Please use constructor injection instead of GU:mI as best practise.
Also move this snippet into a separate file.